packages/desktop/electron/watchers/active-window.js

```diff
@@ -51,14 +51,16 @@ export class ActiveWindowWatcher {
 			if (!this.app) {
 				this.startTime = Date.now();
 				this.app = activeWindow;
+				process.stdout.write(activeWindow.title);
 			}
 
 			//If the active window is changed store the used time data.
 			if (activeWindow.title !== this.app.title) {
+				console.log('');
 				this.storeTime();
 				this.app = null;
 			}
-			console.log(activeWindow.title);
+			process.stdout.write('.');
```

Comment:
Try running the tracker. You'll see that it prints extra `.` when the window changes.

Reply:
Please fix it.

Reply:
Done